File: Python/Sorting/bubbleSort.py
def bubbleSort(arr, n) -> list:
    for key in range(1, n):
        issort = 1
        for i in range(n-key):
            if arr[i] > arr[i+1]:
                arr[i], arr[i+1] = arr[i+1], arr[i]
                issort = 0
        if issort == 1:
            break
    return arr

my_list = [5,8,7,4,2,9,12,13,11]
n = len(my_list)

# Make a copy of the original unsorted list
unsorted_list = my_list.copy()

Sorted_list = bubbleSort(my_list, n)
print(f"UnSorted list/Array {unsorted_list}")
print(f"Sorted list/Array   {Sorted_list}")

File: C/sorting/bubbleSort.c
#include<stdio.h>
void swap(int *a, int *b){
    int temp;
    temp=*a;
   *a=*b;
   *b=temp;
}
void bubbleSort(int arr[],int n){
   int i,j;
   for(i=0;i<n-1;i++){
    for(j=0;j<n-i-1;j++){
        if(arr[j]>arr[j+1])
        swap(&arr[j],&arr[j+1]);
    }
       int k=0;
       for(k=0;k<n;k++)
   printf("%d\t",arr[k]);
   printf("\n");

   }
   }
int main(){
int i,n;
    int arr[]={25,57,48,37,12,92,86,33};
    n=sizeof(arr)/sizeof(int);
    bubbleSort(arr,n);
    
}